child_compassion/models/child_compassion.py

```diff
@@ -537,6 +537,32 @@ def fetch_translations(self):
         self._fetch_translations(self.env.ref("child_compassion.beneficiaries_details"))
         return self.edit_translations()
 
+    @api.model
+    def update_all_children_description(self):
+        children = self.env["compassion.child"].sudo().search([])
```

Comment:
We can avoid updating children that are no longer active, it will be a bit quicker.
```suggestion
        children = self.env["compassion.child"].sudo().search([
            ("state", "not in", ["F", "R"])
        ])
```